src/web/forge_style.css

```diff
@@ -0,0 +1,8 @@
+#projectlogo img {
+    height: 8em;
+}
+
+#titlearea {
+    padding-left: 10px;
```

Comment:
Instead of adding padding (which causes the horizontal scrollbar to appear), could we add a custom id to the version picker and apply a margin to that element instead?